src/elements/content-sidebar/activity-feed/app-activity/AppActivity.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import classNames from 'classnames';
+import getProp from 'lodash/get';
+import noop from 'lodash/noop';
+import TetherComponent from 'react-tether';
+import { FormattedMessage, injectIntl, IntlShape } from 'react-intl';
+
+import ActivityCard from '../ActivityCard';
+import ActivityTimestamp from '../common/activity-timestamp';
+import DeleteConfirmation from '../common/delete-confirmation';
+import IconTrash from '../../../../icons/general/IconTrash';
+import Media from '../../../../components/media';
+import messages from './messages';
+import { bdlGray80 } from '../../../../styles/variables';
+import { Link } from '../../../../components/link';
+import { MenuItem } from '../../../../components/menu';
+import type { AppItem, ActivityTemplateItem, ActionItemError } from '../../../../common/types/feed';
+import type { User, BoxItemPermission } from '../../../../common/types/core';
+import './AppActivity.scss';
+
+interface AppActivityProps {
```

Comment:
```suggestion
export interface AppActivityProps {
```

then remove the exported line at the bottom of the file